src/main/java/com/google/capstone/api/MyResource.java

```diff
@@ -0,0 +1,25 @@
+package com.google.capstone.api;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Root resource (exposed at "myresource" path)
+ */
+@Path("myresource")
+public class MyResource {
```

Comment:
Looks like this is an example endpoint. Do we need to check this in or can we skip it?